lib/controllers/api/spree/v2/storefront/account_controller_decorator.rb

```diff
@@ -10,6 +10,18 @@ def self.prepended(base)
           def create
             result = Spree::Account::Create.call(user_params: spree_user_params)
 
+            render_payload(result)
+          end
+
+          def update
+            result = Spree::Account::Update.call(user: spree_current_user, user_params: spree_user_params)
```

Comment:
@damianlegawiec should I use `spree_current_user` here? I need your confirm

Reply:
@v10110 yes, you need to use `spree_current_user`